gui/src/2D/controls/control.ts

```diff
@@ -1369,6 +1372,41 @@ export class Control {
         return false;
     }
 
+    /** @hidden */
+    private onClipboardCopy = (evt: ClipboardEvent) => {

```

Comment:
Not sure to understand why this has to be here? Why not on ADT?

Reply:
Putting this is ADT results in loss of original triggered event, it somehow bypasses it, though the listener remains active. Because of this the browser's clipboard is not accessible(it can only be accessed by auto triggered/trusted events) 

Will try to research more on this. But even though it is registered at control level, the data is copied to browser's clipboard and remains live even if the control item is disposed.

Reply:
Found the mystery of the era -
```
if (evt instanceof KeyboardEvent) {
         this.processKey(evt.keyCode, evt.key, evt);
          evt.preventDefault(); //this was causing the loss of trigger!!  ohh.. GOD
 } 
```
Moving the registration part to ADT... 👍

Reply:
Aye aye Captain !!! I see, just as scene listens to Keyboard and Mouse events